pulsar-broker/src/main/java/org/apache/pulsar/broker/service/Topic.java

```diff
@@ -338,7 +338,7 @@ default boolean isSystemTopic() {
 
     boolean isPersistent();
 
-    boolean isFenced();
```

Comment:
Should we keep this interface here? It might break some protocol handler compatibility if removed

Reply:
This only introduced for pip-307. Since we dont use it any more, we are deleting it.

Reply:
I see, thanks for explaining.